rust/gui-client/src-tauri/src/client/resolvers.rs

```diff
@@ -7,12 +7,8 @@ mod imp {
     use anyhow::Result;
     use std::net::IpAddr;
 
-    // TODO: The code here will depend on the chosen DNS control method.
-    // So that will need to be threaded in here somehow.
-    #[allow(clippy::unnecessary_wraps)]
     pub fn get() -> Result<Vec<IpAddr>> {
-        tracing::error!("Resolvers module not yet implemented for Linux, returning empty Vec");
-        Ok(Vec::default())
+        firezone_headless_client::imp::get_system_default_resolvers_systemd_resolved()
```

Comment:
I think all desktop systems we support will have systemd and probably systemd-resolved, so I made this the default here.

Reply:
Yep! Makes sense. "Install systemd-resolved" is a valid requirement I think.